src/ydata_profiling/model/summary_algorithms.py

```diff
@@ -2,12 +2,12 @@
 from typing import Any, Callable, Optional, Tuple, TypeVar
 
 import numpy as np
-import pandas as pd
 from multimethod import multimethod
 from scipy.stats import chisquare
-
 from ydata_profiling.config import Settings
 
+import pandas as pd
+

```

Comment:
why the change in the import order?

Reply:
I'm using automatic import sorter. I could change that for this MR